app/views/users/show.html.erb

```diff
@@ -1,9 +1,16 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <div>
+    <h2><%= "Name: #{@user.name}" %></h2>
+   
+    <%= show_send_invitation_button(@user) %>
+    <%= "<b>Pending request</b>".html_safe if @user.inverse_friendships.find{|f| f.user_id == current_user.id and !f.confirmed} %>
+    <%= "<b>Already friend</b>".html_safe if @user.friends.include?(current_user) %>
```

Comment:
- [ ] There should be NO application logic in the views nor helpers (no Post.new, Comment.create, current_user.friends.find_by(...)...). Everything connected with data processing should be encapsulated in models and used in controllers.